modules/dxp/apps/portal-search-tuning/portal-search-tuning-blueprints-admin-web/src/main/resources/META-INF/resources/js/edit_blueprint/ResultListItem.js

```diff
@@ -26,24 +26,25 @@ const RESULTS_DEFAULT_KEYS = [
 	'b_created',
 	'b_modified',
 	'b_author',
-	'b_assetEntryId',
 ];
-const RESULTS_HIDE_KEYS = ['explanation', 'document', 'b_viewURL'];
+const RESULTS_SHOW_KEYS = ['b_assetEntryId', 'id'];
 
 function ResultListItem({item}) {
 	const [collapse, setCollapse] = useState(true);
 
-	const _renderListRow = (property) => (
-		<ClayLayout.Row justify="start" key={`${item.id}_${property}`}>
+	const _renderListRow = (property, value, id) => (
+		<ClayLayout.Row justify="start" key={`${id}_${property}`}>
 			<ClayLayout.Col className="semibold" size={4}>
-				{property}
+				{property.match(/b_[\w_]+/g) ? property.substring(2) : property}
 			</ClayLayout.Col>
 
 			<ClayLayout.Col
 				className={getCN({'text-truncate': collapse})}
 				size={8}
 			>
-				{item[property]}
+				{typeof value === 'object'
+					? JSON.stringify(value).replace(/[[\]]+/g, '')
+					: value}
```

Comment:
Similar to the previous comment, can we extract this functionality?

Reply:
Oh, could you let me know what you think of this? Let me know if we want to define the brackets regex or include object check inside the same function 😅 

<code>
const blueprintBracketsRegex = new RegExp(/[\[\]]/, 'g');

function formatResultObject(value) {
	return JSON.stringify(value).replace(blueprintBracketsRegex, '');
}
</code>

and then for the application:

<code>
{typeof value === 'object'
					? formatResultObject(value)
					: value}
</code>

Reply:
I think just the brackets regex is good enough for now. We also don't need the `blueprint` part of the variable name here either since it's not blueprint specific. `bracketsRegex` would be more appropriate. 

Also I think the function name `removeBrackets` would be a better name that explains what the function is doing. Since an object could also be `{"test": "test"}` and this function wouldn't change the formatting of that.